VCEntities/VCEntities/backup/content/microsoft2020/Microsoft2020IdentifierBackup.swift

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import Foundation
+
+public struct Microsoft2020IdentifierBackup : Codable {
+    
+    private struct Constants {
+        static let OnTheWireMainIdentifier = "did.main.identifier"
```

Comment:
can you write a comment in here that this adheres to Android backup data model, but something we should revisit?